src/auth_service/user_management/claims_repository/core/claims.py

```diff
@@ -69,7 +68,7 @@ def create_data_steward_claim(user_id: str, valid_days=365) -> ClaimFullCreation
     valid_until = now_as_utc() + timedelta(days=valid_days)
     return ClaimFullCreation(
         visa_type=VisaType.GHGA_ROLE,
-        visa_value=parse_obj_as(EmailStr, f"{DATA_STEWARD_ROLE}@{INTERNAL_DOMAIN}"),
+        visa_value=f"{DATA_STEWARD_ROLE}@{INTERNAL_DOMAIN}",
```

Comment:
Optionally, if your input type does not match the type in the model (here str vs EmailStr) you can also use the `model_validate` method to pass in the input data as dict:
```
ClaimFullCreation.model_validate({"visa_value": f"{DATA_STEWARD_ROLE}@{INTERNAL_DOMAIN}", ...})
```
This way you won't get any editor support on which parameters are required, however, at least pyright will not complain.

Reply:
Good idea. In this case it's not needed because I don't get a Pyright error here. I think it's because EmailStr is a subclass of str. It may be usedfull for URLs which are not a subclass of str in Pydantic v2.